server/index.js

```diff
@@ -28,7 +20,13 @@ const mailerRouter = require('./routes/mailer');
 const activitiesRouter = require('./routes/activity');
 const unlockLocks = require('./common/utils/unlockLocks');
 
-/* eslint-enable import/order */
+const app = express();
+const server = http.Server(app);
+
+socket.init(server);
+const socketInstance = socket.getSocketInstance(server);
+socket.listen(socketInstance);
```

Comment:
proposal: could we move line 27-28 to socket `index.js` file and initialize socket listeners in `init` function? , this index app will be cleaner?